python/Dynamic_Scripts/sta_ddos.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/env python3
+"""SecureX Traffic Analytics tool to dynamically measure the rate of change of
```

Comment:
This repo has higher visibility than the learning labs repo on github. Because of this, we need to keep the new name secret until it is publicly announced in 2 weeks